### PR TITLE
docs: added snapshot plugin 

### PR DESCRIPTION
**Pull request by AJDatacore**

### CODE REVIEW

File: reference/kubectl-plugin.md [hunk 1]
```diff
@@ -58,9 +63,9 @@ SUBCOMMANDS:
     uncordon    'Uncordon' resources
 ```
 
-The plugin can be used to get the following resource information:
+You can use the following plugins:
```
Comment (avishnu) [inline reference/kubectl-plugin.md]: Seems incorrect as this needs to mention the various resources.

Comment (avishnu) [reply reference/kubectl-plugin.md]: > Seems incorrect as this needs to mention the various resources.

Ok, when I checked below, we seem to be clubbing the various options. If that's the intent, then the text can be "you can use the plugin with the following options or in the following way" ?

File: reference/kubectl-plugin.md [hunk 2]
```diff
@@ -101,7 +106,7 @@ mayastor-pool-3  5360320512      3258974208     aio:///dev/vdb?uuid=f324edb7-1ac
 {% endtab %}
 {% endtabs %}
 
-- **Nodes**
+### Get Maystore Nodes
```
Comment (tiagolobocastro) [inline reference/kubectl-plugin.md]:
```suggestion
### Get Mayastor Nodes
```

File: reference/kubectl-plugin.md [hunk 3]
```diff
@@ -144,7 +149,7 @@ Volume 0c08667c-8b59-4d11-9192-b54e27e0ce0f Scaled Successfully 🚀
 {% endtabs %}
 
 
-- Retrieving resource specs in any desired format
+### Retrieve resource specs in any desired format
```
Comment (tiagolobocastro) [inline reference/kubectl-plugin.md]:
```suggestion
### Retrieve resource in any desired output format
```
I'm not sure here **any** desired format is correct, we only support 3.

**Review by Abhinandan-Purkait — APPROVED** [review]

File: reference/kubectl-plugin.md [hunk 4]
```diff
@@ -144,7 +149,7 @@ Volume 0c08667c-8b59-4d11-9192-b54e27e0ce0f Scaled Successfully 🚀
 {% endtabs %}
 
 
-- Retrieving resource specs in any desired format
+### Retrieve resource in any output format
```
Comment (avishnu) [inline reference/kubectl-plugin.md]: Retrieve resource details in the output formats table, json or yaml, with table being the default output format

**Review by avishnu — APPROVED** [review]
LGTM

**Pull request merged**